### PR TITLE
Use the multiversx-sdk python package & update cookbook

### PR DESCRIPTION
**Pull request by popenta**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [x] document new feature
- [x] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-py/sdk-py.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ This SDK consists of a CLI tool, namely `mxpy`, and Python helpers and utilities
...
+| [multiversx-sdk](https://pypi.org/project/multiversx-sdk)                           | [mx-sdk-py](https://github.com/multiversx/mx-sdk-py)            | The all in one Python SDK for interacting with MultiversX. It's the unification of the previous packages (multiversx-sdk-core, multiversx-sdk-wallet and multiversx-sdk-network-providers). |
+| [multiversx-sdk-core](https://pypi.org/project/multiversx-sdk-core)                           | [mx-sdk-py-core](https://github.com/multiversx/mx-sdk-py-core)            | Basic components for interacting with the blockchain and with smart contracts. Will soon be deprecated. |
+| [multiversx-sdk-wallet](https://pypi.org/project/multiversx-sdk-wallet)                       | [mx-sdk-py-wallet](https://github.com/multiversx/mx-sdk-py-wallet)                       | Core wallet components (generation, signing). Will soon be deprecated. |
+| [multiversx-sdk-network-providers](https://pypi.org/project/multiversx-sdk-network-providers) | [mx-sdk-py-network-providers](https://github.com/multiversx/mx-sdk-py-network-providers) | Network providers (API, Gateway). Will soon be deprecated. |
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/sdk-py.md:39]: `Will soon be deprecated.` should be in bold.

Comment (andreibancioiu) [reply docs/sdk-and-tools/sdk-py/sdk-py.md:39]: Also, perhaps add an extra `:::note ... :::` in the section - an additional heads-up on using the unified library.

File: docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md [hunk 2]
```diff
@@ -236,34 +227,53 @@ transaction = transfer_factory.create_transaction_for_esdt_token_transfer(
...
-```
+```py
 from pathlib import Path
-from multiversx_sdk_core.transaction_builders.relayed_v1_builder import RelayedTransactionV1Builder
-from multiversx_sdk_core import Transaction, TransactionComputer, Address
-from multiversx_sdk_wallet.user_signer import UserSigner
 
-signer = UserSigner.from_pem_file(Path("./testwallets/bob.pem"))
+from multiversx_sdk.core import Address, Transaction, TransactionComputer
+from multiversx_sdk.core.transactions_factories.relayed_transactions_factory import \
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md:270]: Can we have simpler imports here?

File: docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md [hunk 3]
```diff
@@ -276,26 +286,35 @@ inner_tx = Transaction(
...
-from multiversx_sdk_core.transaction_builders.relayed_v2_builder import RelayedTransactionV2Builder
-from multiversx_sdk_wallet.user_signer import UserSigner
+```py
+from pathlib import Path
+
+from multiversx_sdk.core import Address, Transaction, TransactionComputer
+from multiversx_sdk.core.transactions_factories.relayed_transactions_factory import \
+    RelayedTransactionsFactory
+from multiversx_sdk.core.transactions_factories.transactions_factory_config import \
+    TransactionsFactoryConfig
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md:311]: Can we have simpler imports?

File: docs/sdk-and-tools/sdk-py/sdk-py.md (hunk 1 above)
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/sdk-py.md]: Added

File: docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md (hunk 2 above)
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md]: Regenerated the cookbook. Imports look like this:
```py
from multiversx_sdk import ...
```

File: docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md (hunk 3 above)
Comment (popenta) [reply docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md]: Done

**Review by andreibancioiu — DISMISSED** [review]

File: docs/sdk-and-tools/sdk-py/sdk-py.md [hunk 4]
```diff
@@ -31,8 +31,13 @@ This SDK consists of a CLI tool, namely `mxpy`, and Python helpers and utilities
 
 ## `sdk-py`: the Python libraries
 
+:::note
+The `multiversx-sdk` package is a unification of the previous packages (multiversx-sdk-core, multiversx-sdk-wallet and multiversx-sdk-network-providers). This is the package that **should** be used from now on. All the upcoming updates will come only to this package. The older packages will no longer receiver updates and will soon be deprecated.
```
Comment (bogdan-rosianu) [inline docs/sdk-and-tools/sdk-py/sdk-py.md:35]: will no longer receive

Comment (popenta) [reply docs/sdk-and-tools/sdk-py/sdk-py.md]: fixed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**